ts/Extensions/ScrollablePlotArea.ts

```diff
@@ -177,7 +177,11 @@ class ScrollablePlotArea {
                 for (const axis of chart.axes) {
                     // Apply the corrected plot size to the axes of the other
                     // orientation than the scrolling direction
-                    if (axis.horiz === recalculateHoriz) {
+                    if (
+                        axis.horiz === recalculateHoriz ||
+                        // Or parallel axes
+                        (chart.hasParallelCoordinates && !axis.horiz)
```

Comment:
Horizontal axes can also be ||-coords y-axes: https://jsfiddle.net/highcharts/hcxpbrqe/ . It seems to me that the following is more precise?
```suggestion
                        (chart.hasParallelCoordinates && axis.coll === 'yAxis')
```